ios/brave-ios/Sources/Brave/Frontend/Passcode/WindowProtection.swift

```diff
@@ -86,18 +86,16 @@ public class WindowProtection {
   private let lockedViewController = LockedViewController()
 
   private var cancellables: Set<AnyCancellable> = []
-  private var protectedWindow: UIWindow
   private var passcodeWindow: UIWindow
   private var context = LAContext()
   private var viewType: AuthViewType = .general
 
   private var isVisible: Bool = false {
     didSet {
       passcodeWindow.isHidden = !isVisible
+      // Prior window will be made key automatically after passcode window is dismissed
       if isVisible {
         passcodeWindow.makeKeyAndVisible()
-      } else {
-        protectedWindow.makeKeyAndVisible()
```

Comment:
First of all I tested the change both for youtube problem and when passcode is enabled etc

But the part I dont understand, dont we have to set active window with makeKeyAndVisible when passcode is dismissed ?

Reply:
In my testing I found that It doesn't seem to be required, upon the `passcodeWindow` being hidden it simply gets removed from the window list and the last key window before showing it becomes key again